site/gdocs/components/centered-article.scss

```diff
@@ -1072,6 +1072,62 @@ div.raw-html-table__container {
     }
 }
 
+.article-block__cookie-notice {
+    .cookie-preferences {
+        margin: 0;
+        padding: 32px 24px;
+        border: none;
+        background-color: $blue-5;
+        h2 {
```

Comment:
The convention I'm familiar with, which we mostly(?) follow, is to put a blank line between blocks of CSS, before a new selector starts. I'd find it more readable if we could do that, but feel free to keep it as is, if you have the opposite preference. 🙂

Reply:
Too true! My subconscious wanted to minimize the amount of overrides I had to write to get this to look right 😭